app/models/course.rb

```diff
@@ -13,6 +15,11 @@ class Course < ActiveRecord::Base
   has_many :announcements, inverse_of: :course, dependent: :destroy
   has_many :achievements, inverse_of: :course, dependent: :destroy
 
+  # @return [Array] array of all available modules
+  def modules
+    ActsAsTenant.current_tenant.enabled_modules
```

Comment:
Maybe we should extract this out to be an Instance method...? `Instance.current.enabled_modules`